frontend/router/page_titles.ts

```diff
@@ -6,67 +6,31 @@ import PATHS from "router/paths";
 // Note: Order matters for use of array.find() (specific subpaths must be listed before their parent path)
 export default [
   { path: PATHS.DASHBOARD, title: `Dashboard | ${DOCUMENT_TITLE_SUFFIX}` },
-  { path: "/hosts/manage", title: `Manage hosts | ${DOCUMENT_TITLE_SUFFIX}` },
+  { path: "/hosts/manage", title: `Hosts | ${DOCUMENT_TITLE_SUFFIX}` },
   {
-    path: PATHS.CONTROLS_OS_UPDATES,
-    title: `Manage OS updates | ${DOCUMENT_TITLE_SUFFIX}`,
+    path: "/controls",
```

Comment:
Wherever possible, we're using paths as defined in `paths.ts` to keep that single source of truth for page routes. This prevents bugs if, for example, we ever update the `controls` route, we would only have to update it there (and not here). Can you please use `PATHS.CONTROLS` here.

Reply:
totes, will do after our meeting!